packages/executors/src/serverExecutor.ts

```diff
@@ -71,20 +72,18 @@ export class ServerExecutor implements IExecutor {
     this.token = this.retrieveSSEToken();
   }
 
-  private initializeSocket() {
+  private async initializeSocket() {
     if (!this.sandboxId) {
       throw new Error('initializeSocket: sandboxId is not defined');
     }
-
     const usedHost = this.host || 'https://codesandbox.io';
-    const sseHost = usedHost.replace('https://', 'https://sse.');
+    const sseLbHost = usedHost.replace('https://', 'https://sse-lb.');
+    const res = await axios.get(`${sseLbHost}/api/cluster/${this.sandboxId}`);
```

Comment:
Maybe we should put this in a function that retries up to 3 times, in case of flaky connections or when the LB has an error.

Reply:
That's a good point, do we use any retry library anywhere that we could reuse here? Also I think some error handling would be nice, in case all retries fail, but I'm not sure what the UI/UX pattern for this is.

Reply:
We don't use anything generic right now, I have written a poor mans retry function. Would be good to introduce some common helper function in `@codesandbox/common` or a library I think.

Regarding the notice, we can send a notification from here. You can import
```
import {
  notificationState,
  NotificationStatus
} from '@codesandbox/common/lib/utils/notifications';
```

and do

```
notificationState.addNotification({ message: 'blah', status: NotificationStatus.Error })
```